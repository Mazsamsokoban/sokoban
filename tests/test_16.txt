loadtest 16
showstate w
showstate box
showstate shole
move w right
showmaze
showstate w
showstate box
showstate shole