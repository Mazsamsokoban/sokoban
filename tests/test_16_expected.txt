f1:w f2:box shole:null 

name:w
field:f1
direction:Neutral
points:0
pushforce:10.0

name:box
field:f2
direction:Neutral
owner:null
pusher:null

name:shole
friction:Normal
thing:null
open:false

f1:null f2:w shole:box 

name:w
field:f2
direction:Right
points:0
pushforce:10.0

name:box
field:shole
direction:Right
owner:null
pusher:w

name:shole
friction:Normal
thing:box
open:false

